packages/generator/src/vdm-types.ts

```diff
@@ -270,7 +270,7 @@ export interface VdmComplexType {
 /**
  * @internal
  */
-export interface VdmFunctionImportBase {
+export interface VdmOperationBase {
```

Comment:
[pp] Since the name is now not function or action consider adding a small JS doc to explain what this represents.